smart_campus/app/templates/app/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Please leverage the power of Django template.
Move common HTML code to a "base.html" and import it.